### DISCUSSION TITLE
GraphiQL IDE mutation error after updating to version above 1.6.12

### DISCUSSION THREAD

**Discussion by alexostar** [wp-graphql/wp-graphql#2334] [General] [answered]
Hi, I'm relatively new to WPGraphQL. After I updated from 1.6.12, my mutations (on a CPT called "Notes") do not work in the GraphiQL IDE but I get the following error message; "Sorry, you are not allowed to create Notes". When I return to 1.6.12 everything works fine again. Is there somehting I should do differently in 1.7 and above. In the newer version there is $ sign in the Explorer directly after "input", before the list of fields, which is not there in 1.6.12, so I assume I'm supposed to add something there. 
By the way, this happens only when using the GraphiQL IDE on the WordPress site. Using the same mutation in the NextJs app that I'm working on works fine, so this is not a serious problem for me, but it makes testing more difficult. Thanks in advance for your help.

**Comment by justlevine** [comment, marked as answer]
>I get the following error message; "Sorry, you are not allowed to create Notes". 

This mean's the user trying to create the post doesnt have the correct auth.

> By the way, this happens only when using the GraphiQL IDE on the WordPress site. Using the same mutation in the NextJs app that I'm working on works fine.

1.7.0 added a toggle in GraphiQL to switch between an authed/unauthed user. Can you confirm you're set as authed? (note the green dot on the top right of the user avatar.
![image](https://user-images.githubusercontent.com/29322304/162580672-a7fdd7bb-a65a-435f-9bd7-147377e39e33.png)

**Reply by alexostar** [reply]
That solved the problem! I had missed noticing this new toggle feature. Thank you for the quick response, - you saved my weekend:)

**Reply by Starlette** [reply]
I am having this issue but I need to be able to create the post publicly (custom post type for users asking questions).

**Reply by justlevine** [reply]
@Starlette WPGraphQL respects WordPress's core permissions model. If you want posts to be publicly creatable you can set the `capabilities` on when [registering the post type](https://developer.wordpress.org/reference/functions/register_post_type/#parameters).
